Realm/Realm/Handles/SessionHandle.cs

```diff
@@ -405,10 +405,12 @@ private static IntPtr NotifyAfterClientReset(IntPtr beforeFrozen, IntPtr after,
         }
 
         [MonoPInvokeCallback(typeof(NativeMethods.SessionProgressCallback))]
-        private static void HandleSessionProgress(IntPtr tokenPtr, ulong transferredBytes, ulong transferableBytes)
+        private static void HandleSessionProgress(IntPtr tokenPtr, ulong transferredBytes, ulong transferableBytes, double progressEstimate)
         {
             var token = (ProgressNotificationToken?)GCHandle.FromIntPtr(tokenPtr).Target;
-            token?.Notify(transferredBytes, transferableBytes);
+            // This is used to provide a reasonable progress estimate until the core work is done
```

Comment:
Single-line comment should be preceded by blank line ([`https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1515.md`](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1515.md))

😛